src/components/features/PromoBox/PromoBox.module.scss

```diff
@@ -91,4 +91,42 @@
   padding: 0;
 }
 
+.promoDescription {
+  p:first-of-type {
+    span {
+      font-weight: bold;
+    }
+    font-size: 4rem;
+    margin: 0;
+  }
+  p:last-of-type {
+    font-size: 1.5rem;
+  }
+  position: absolute;
+  padding-top: 7%;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: $color-light-primary;
+  text-transform: uppercase;
+  background: $background-transparent;
+  width: 100%;
+  height: 14rem;
+  text-align: center;
+}
+
+.shopButton {
+  position: absolute;
+  top: 67%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Zastanawiam się czy konieczne jest wyciągnaie buttona do position absolute? Ja bym go zostawił  divie promoDescription i ułozył flexem.

Reply:
dobra, jak ma wychodzić poza półprzeźroczysty pasek to trzeba tak.

Reply:
Czyli mam rozumieć że problemu nie ma? :D

Reply:
jest :D Napisałem wyżej, umieść przyciski wewnatrz promoDescription